### PR TITLE
Clean create initiative

### PR DESCRIPTION
**Pull request by iamlasse**
Clean up to include initiative limit

### CODE REVIEW

File: client/templates/initiative/create/initiativeCreate.js [hunk 1]
```diff
@@ -2,6 +2,13 @@ Template.initiativeCreate.onRendered(function() {
   $('select').material_select();
 });
 
+Template.initiativeCreate.helpers({
+  overLimit: function(){
+    var initiatives = Initiatives.find({createdBy: Meteor.userId()}).count();
```
Comment (trustieee) [inline client/templates/initiative/create/initiativeCreate.js:7]: Can you test this on a user with no Initiatives? Will it crash the client (calling .count() on an undefined object if mongo returns undefined for no matches?)

Comment (iamlasse) [inline client/templates/initiative/create/initiativeCreate.js:7]: I created a new user and didnt bark at it...

Comment (iamlasse) [inline client/templates/initiative/create/initiativeCreate.js:7]: did it again, no problem

Comment (trustieee) [inline client/templates/initiative/create/initiativeCreate.js:7]: i tested as well with meteor mongo, looks good.

**Pull request merged**